src/Sentry.Unity/SentryUnity.cs

```diff
@@ -32,7 +32,9 @@ public static void Init(SentryUnityOptions unitySentryOptions)
             // Uses the game `version` as Release unless the user defined one via the Options
             if (unitySentryOptions.Release == null)
             {
-                unitySentryOptions.Release = $"{Application.productName}@{Application.version}";
+                unitySentryOptions.Release = String.IsNullOrWhiteSpace(Application.productName)
```

Comment:
`String` can bind to another class, while `string` is an alias to `System.String`. Use `string` instead of `String`.